packages/dashmate/src/ssl/generateKeyPair.js

```diff
@@ -0,0 +1,16 @@
+const forge = require('node-forge');
```

Comment:
forge is outdated and very slow (not suitable for Raspberry Pi and such). Consider using the node crypto and acme:

- https://nodejs.org/api/crypto.html
- https://www.npmjs.com/package/acme

These are fast, lightweight, and use native crypto.

Reply:
Good point, thank you. @shuplenkov please update it your current PR.